velox/docs/functions/spark/aggregate.rst

```diff
@@ -12,6 +12,7 @@ General Aggregate Functions
     Returns the last value of `x` for a group of rows.
 
 .. spark:function:: last(x, ignoreNull) -> x
-
+    :noindex:
```

Comment:
why is there noindex on certian functions ?

Reply:
Thanks, just remove it.